[2023-06-27T14:59:02.366+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice permission to 509
[2023-06-27T14:59:02.367+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice'
[2023-06-27T14:59:02.399+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice permission to 509
[2023-06-27T14:59:02.400+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice'
[2023-06-27T14:59:02.541+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.get_choice scheduled__2023-06-27T14:58:00+00:00 [queued]>
[2023-06-27T14:59:02.550+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 2_Test_Badge_5_Loop.get_choice scheduled__2023-06-27T14:58:00+00:00 [queued]>
[2023-06-27T14:59:02.551+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-06-27T14:59:02.566+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): get_choice> on 2023-06-27 14:58:00+00:00
[2023-06-27T14:59:02.571+0000] {standard_task_runner.py:57} INFO - Started process 8195 to run task
[2023-06-27T14:59:02.575+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', '2_Test_Badge_5_Loop', 'get_choice', 'scheduled__2023-06-27T14:58:00+00:00', '--job-id', '371', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpi42f0fhi']
[2023-06-27T14:59:02.577+0000] {standard_task_runner.py:85} INFO - Job 371: Subtask get_choice
[2023-06-27T14:59:02.624+0000] {logging_mixin.py:149} INFO - Changing /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice permission to 509
[2023-06-27T14:59:02.625+0000] {logging_mixin.py:149} INFO - Failed to change /opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=2_Test_Badge_5_Loop/run_id=scheduled__2023-06-27T14:58:00+00:00/task_id=get_choice'
[2023-06-27T14:59:02.628+0000] {task_command.py:410} INFO - Running <TaskInstance: 2_Test_Badge_5_Loop.get_choice scheduled__2023-06-27T14:58:00+00:00 [running]> on host eb22b15bae36
[2023-06-27T14:59:02.751+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='2_Test_Badge_5_Loop' AIRFLOW_CTX_TASK_ID='get_choice' AIRFLOW_CTX_EXECUTION_DATE='2023-06-27T14:58:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-27T14:58:00+00:00'
[2023-06-27T14:59:02.753+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-06-27T14:59:02.754+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'cd /dbt && dbt run-operation get_stream_value --args \'{                            "stream_name": "ed_cdc_stream",                             "dry_run": false}\'']
[2023-06-27T14:59:02.764+0000] {subprocess.py:86} INFO - Output:
[2023-06-27T14:59:05.824+0000] {subprocess.py:93} INFO - [0m14:59:05  Running with dbt=1.5.1
[2023-06-27T14:59:12.355+0000] {subprocess.py:93} INFO - [0m14:59:12  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2023-06-27T14:59:12.356+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2023-06-27T14:59:12.357+0000] {subprocess.py:93} INFO - - models.badge_5.time_of_day_lu
[2023-06-27T14:59:12.400+0000] {subprocess.py:93} INFO - [0m14:59:12  Found 3 models, 0 tests, 0 snapshots, 0 analyses, 456 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics, 0 groups
[2023-06-27T14:59:13.682+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-06-27T14:59:13.712+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=2_Test_Badge_5_Loop, task_id=get_choice, execution_date=20230627T145800, start_date=20230627T145902, end_date=20230627T145913
[2023-06-27T14:59:13.787+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-06-27T14:59:13.817+0000] {taskinstance.py:2651} INFO - 1 downstream tasks scheduled from follow-on schedule check
